### ISSUE TITLE
TikTok's been working great, but there's a new issue

### ISSUE THREAD

**Issue by spinningsand** [go-olive/olive#40]
TikTok's been working great for the past 8 weeks or so! But today, I noticed it wasn't taking a snap of someone who was live -- it just kept skipping over them and going to the next streamer. So then I shut olive down and started it over and when it gets to the streamer it skipped, it says `Started polling` and stops there until they go offline.  It doesn't take my any snaps of anyone while they're still on.

EDIT: So it turns out that it does continue with snaps, but only after they stopped polling. If someone is running it poll, it pauses until they're done.

**Comment by luxcgo** [comment]
Yes, it is exactly how it is implemented under the hood. If it's already recording, there's no need to monitor anyways. At the same time it saves traffic. (Btw, the previous version was implemented as you expect.

**Comment by spinningsand** [comment]
It doesn't record anyone anymore. Anytime someone is live, all it says is `Started polling`

**Comment by spinningsand** [comment]
Alright, so I didn't have the latest version and when I uninstalled and re-installed the latest version, it worked again. Maybe I should've tried that first before opening a new issue lol.

But since I started a new issue -- I was wondering -- did the IP blocking issue with TikTok ever get fixed? I haven't tried it since before, so I'm not certain.

**Comment by luxcgo** [comment]
actually no, since it may requires additional ips

**Comment by spinningsand** [comment]
So I have 1 more question. How exactly do I use `-u` for tiktok? I tried was using the URL address, but it didn't work for me.

**Comment by luxcgo** [comment]
It's kind of a bug. Tiktok's id is a bit different than others. 
Remove the `@` character in url and try again.

**Comment by spinningsand** [comment]
Thanks! That helped.

**Comment by spinningsand** [comment]
Do you know how to go about fixing the IP blocking? I've had my IP get blocked at random several times. It'll get blocked for a day or 2 and then it's fine. It also can get blocked if I use `-u` .
